src/machines/workflow_machines.rs

```diff
@@ -536,7 +540,11 @@ impl WorkflowMachines {
         Ok(())
     }
 
-    fn handle_driven_results(&mut self, results: Vec<WFCommand>) -> Result<()> {
+    fn handle_driven_results(
```

Comment:
This dude probably deserves a docstring now

Reply:
Added
   /// Handles results of the workflow activation, delegating work to the appropriate state machine.
    /// Returns a list of workflow jobs that should be queued in the pending activation for the next poll.
    /// This list will be populated only if state machine produced lang activations as part of command processing.
    /// For example some types of activity cancellation need to immediately unblock lang side without
    /// having it to poll for an actual workflow task from the server.